src/snowflake/snowpark/_internal/compiler/large_query_breakdown.py

```diff
@@ -477,6 +477,14 @@ def _is_relaxed_pipeline_breaker(self, node: LogicalPlan) -> bool:
         if isinstance(node, SelectStatement):
             return True
 
+        if isinstance(node, SnowflakePlan):
+            return node.source_plan is not None and self._is_relaxed_pipeline_breaker(
```

Comment:
What's the difference between "relaxed pipeline breaker" and "pipeline breaker"?

Reply:
pipeline breaker are all nodes like sort, pivot etc which are listed in the is_pipeline_breaker function.
relaxed pipeline breaker are those nodes which are not pipeline breakers but can be used to cut if no valid pipeline breaker is found. For now `SelectStatement` is the only relaxed pipeline breaker.